server/game/triggeredability.js

```diff
@@ -97,6 +97,10 @@ class TriggeredAbility extends BaseAbility {
         return false;
     }
 
+    isPlayableEventAbility() {
```

Comment:
bothers me slightly that this is duplicated, but probably not enough to suggest refactoring these two classes to share a common base